src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -75,16 +95,20 @@ describe('test data', () => {
     cy.wrap(data).its('length').should('be.lte', 50);
   });
   it('has more than one enabled clusters with "custom" in name', () => {
-    cy.wrap(filterData('custom')).its('length').should('be.gt', 1);
+    cy.wrap(filterData({ name: 'custom' }))
+      .its('length')
+      .should('be.gt', 1);
```

Comment:
Don't you want to use the `expect` notation here too already?

Reply:
yes, but as it was part of another MR, I didn't want to merge concepts here. I will merge with the other and update everything.